docs/HowToFAQ/FAQMethodSequence.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Is there a way to write `if foo().equals(bar()) { x } else { y }` where `foo` and `bar` are methods?
+---
+
+## Question
+
+Is there a way to write `if foo().equals(bar()) { x } else { y }` where `foo` and `bar` are methods?
+
+## Answer
+
+No. Methods have side-effects and so Dafny does not allow them in expressions.
```

Comment:
We should explain what an expression and a statement are, here.

Reply:
Expanded the answer